central/image/datastore/store/common/v2/split.go

```diff
@@ -67,6 +87,19 @@ func splitCVEs(parts ImageParts, component ComponentParts, embedded *storage.Emb
 	return ret
 }
 
+func splitCVEsV2(parts ImageParts, component ComponentParts, embedded *storage.EmbeddedImageScanComponent) []CVEParts {
```

Comment:
nit: Can avoid passing `ImageParts` here because component already has the operating system.